src/NodeAutoCompleteViewExtension/NodeAutoCompleteViewModel.cs

```diff
@@ -0,0 +1,359 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Web;
+using System.Windows.Documents;
+using Dynamo.Core;
+using Dynamo.Engine;
+using Dynamo.Graph.Connectors;
+using Dynamo.Graph.Nodes.ZeroTouch;
+using Dynamo.Graph.Nodes;
+using Dynamo.Logging;
+using Dynamo.Models;
+using Dynamo.PackageManager;
+using Dynamo.Search.SearchElements;
+using Dynamo.Utilities;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.ViewModels;
+using Greg;
+using Newtonsoft.Json;
+using RestSharp;
+using System.Windows;
+
+namespace Dynamo.NodeAutoComplete
+{
+    public class NodeAutoCompleteViewModel : NotificationObject
+    {
+        internal UIElement DynamoView { get; set; }
+        internal DynamoViewModel dynamoViewModel { get; set; }
+
+        //constructor
+        internal NodeAutoCompleteViewModel(UIElement dynamoView, DynamoViewModel dvm)
+        {
+            DynamoView = dynamoView;
+            dynamoViewModel = dvm;
+        }
+
+        /*internal MLNodeAutoCompletionRequest GenerateRequestForMLAutocomplete()
```

Comment:
@reddyashish What are these code commented out for?

Reply:
Forgot to remove it. It is duplicated code.